src/state/shellscripts/bwrap.sh

```diff
@@ -11,6 +11,13 @@ if ! command -v bwrap >/dev/null; then
     exit 10
 fi
 
+OPAMVERSION=`opam --version`
```

Comment:
```suggestion
local OPAMVERSION=`opam --version`
```

Reply:
uuh... calling opam from here ??
Also we have no guarantee that the calling process is actually the same version as `opam` looked up in the PATH here ?